src/wui/CMakeLists.txt

```diff
@@ -52,6 +52,7 @@ wl_library(wui_economy_options
     notifications
     io_profile
     ui_basic
+    wui
```

Comment:
Add a note about the cyclic dependency

Reply:
Alternatively you could avoid the cyclic dependency by passing only the EGBase& as ctor parameter as this is the only IBase property the economy options window needs